### ISSUE TITLE
Makefile does not work on make

### ISSUE THREAD

**Issue by rabernethy** [rabernethy/SalesmanGame#4]
make does not work. here is the command sequence I entered:
```bash
Russell:SalesmanGame rabernethy$ make run
make: *** No rule to make target `obj/main.o', needed by `main'.  Stop.
```

**Comment by invention64** [comment]
Think I squashed this one. Left out build instructions for test.o and main.o

**Comment by invention64** [comment]
Issue fixed with pull request #5.
